src/iidm/NodeBreakerVoltageLevel.hpp

```diff
@@ -70,6 +75,8 @@ class NodeBreakerVoltageLevel : public VoltageLevel {
 
     bool isConnected(const Terminal& terminal) const;
 
+    void removeInternalConnections(unsigned long node1, unsigned long node2);
```

Comment:
This method is not well placed: it should not be in the public API. In java, this method is in the different views. If this method is used by the view, as the view is a friend class, move this class in the private part